# Independent HSV Filter

## HSV param
h_min: 150
s_min: 80
v_min: 100
h_max: 16
s_max: 255
v_max: 255

## Laser param
first_detection_depth_thre: 9.0